Gamma_Analysis.py

```diff
@@ -0,0 +1,373 @@
+    elif isotope.Symbol == 'Tl' and isotope.Mass_number == 208:
+        Reference_Conc = Reference.Ref_Concentration[4]
+        Reference_Conc_Unc = Reference.Ref_Concentration_Error[4]
+        Conversion = Reference.Conversion[2]
+    elif isotope.Symbol == 'Ac' and isotope.Mass_number == 228:
+        Reference_Conc = Reference.Ref_Concentration[5]
+        Reference_Conc_Unc = Reference.Ref_Concentration_Error[5]
+        Conversion = Reference.Conversion[2]
+    else:
+        Reference_Conc = 1
+        Reference_Conc_Unc = 0
+        Conversion = 1
+    Ref_Specific_Activity = Reference_Activity[0]/Reference.Mass
+    Ref_Conc_SpecAct_Ratio = Reference_Conc/Ref_Specific_Activity
+    Error_Factor = ((Sample_Activity[1]/Sample_Activity[0])**2 +
+                    (Reference_Activity[1]/Reference_Activity[0])**2 +
+                    (Reference_Conc_Unc/Reference_Conc)**2)**0.5
+    Sample_Factor = Sample_Activity[0]*Ref_Conc_SpecAct_Ratio
+    Sample_Factor_Uncertainty = Sample_Factor*Error_Factor
+    Sample_Concentration = Sample_Factor*Conversion
+    Sample_Concentration_Uncertainty = Sample_Factor_Uncertainty*Conversion
+    Results = [Sample_Concentration, Sample_Concentration_Uncertainty]
+    return Results
+
+
+def peak_finder(Spectrum, Energy):
+    '''
+    PEAK_FINDER will search for peaks within a certain range determined by the
+    Energy given. It takes a Spectra file and an Energy value as input. The
+    energy range to look in is given by the Full-Width-Half-Maximum (FWHM).
+    If more than one peak is found in the given range, the peak with the
+    highest amount of counts will be used.
+    '''
+    E0 = Spectrum.energy_cal[0]
+    Eslope = Spectrum.energy_cal[1]
+    Energy_Axis = E0 + Eslope*Spectrum.channel
+
+    Peak_Energy = []
+    # Rough estimate of FWHM.
+    FWHM = 0.05*Energy**0.5
+
+    # Peak Gross Area
+
+    start_region = np.flatnonzero(Energy_Axis > Energy - 3*FWHM)[0]
+
+    end_region = np.flatnonzero(Energy_Axis > Energy + 3*FWHM)[0]
+    y = Spectrum.data[start_region:end_region]
+    indexes = peakutils.indexes(y, thres=0.5, min_dist=4)
+    Tallest_Peak = []
+    if indexes.size == 0:
+        Peak_Energy.append(int((end_region-start_region)/2)+start_region)
+    else:
+        for i in range(indexes.size):
+            Spot = Spectrum.data[indexes[i]+start_region]
+            Tallest_Peak.append(Spot)
+        indexes = indexes[np.argmax(Tallest_Peak)]
+        Peak_Energy.append(int(indexes+start_region))
+    Peak_Energy = float(Energy_Axis[Peak_Energy])
+    return(Peak_Energy)
+
+
+def peak_measurement(M, energy):
+    """
+    Takes in a measured spectra alongside a specific energy and returns the net
+    area and uncertainty for that energy.
+    """
+    E0 = M.energy_cal[0]
+    Eslope = M.energy_cal[1]
+    energy_axis = E0 + Eslope*M.channel
+    M_counts = M.data
+
+    # Rough estimate of FWHM.
+    FWHM = 0.05*energy**0.5
+
+    # Peak Gross Area
+
```

Comment:
I think this function would be clearer, as well as more compact, if it had fewer blank lines.
Try only putting blank lines between the sections. Like:
```
# Peak Gross Area
start_peak = ...
end_peak = ...
gross_counts_peak = ...

# Left Gross Area
left_peak = ...
...
```